### PR TITLE
add makefile commands for detached cluster manager

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

Relevant Issue: https://github.com/open-cluster-management-io/registration-operator/issues/157

### CODE REVIEW

**Comment by xuezhaojun** [comment]
This PR is only for the makefile commands part. 

/assign @zhujian7 
/assign @zhiweiyin318

File: Makefile [hunk 1]
```diff
@@ -124,6 +135,11 @@ bootstrap-secret-detached:
 	$(KUBECTL) get ns klusterlet; if [ $$? -ne 0 ] ; then $(KUBECTL) create ns klusterlet; fi
 	$(KUSTOMIZE) build deploy/klusterlet/config/samples/bootstrap | $(SED_CMD) -e "s,namespace: open-cluster-management-agent,namespace: klusterlet," | $(KUBECTL) apply -f -
 
+external-hub-secret:
+	cp $(EXTERNAL_HUB_KUBECONFIG) deploy/cluster-manager/config/samples/cluster-manager/external-hub-kubeconfig
+	$(KUBECTL) get ns $(DETACHED_CLUSTER_MANAGER_NAME); if [ $$? -ne 0 ] ; then $(KUBECTL) create ns $(DETACHED_CLUSTER_MANAGER_NAME); fi
+	$(KUSTOMIZE) build deploy/cluster-manager/config/samples/cluster-manager | $(KUBECTL) apply -f -
```
Comment (zhujian7) [inline Makefile]: How to ensure that this secret will be created under the current namespace instead of the fixed `cluster-manager` namespace?

Comment (xuezhaojun) [reply Makefile]: That is a good point! I added a sed replace between `KUSTOMIZE` and `KUBECTL` in the final line of external-hub-secret, please have a look again,

**Comment by zhujian7** [comment]
LGTM

**Pull request merged**